govcd/extension.go

```diff
@@ -5,15 +5,16 @@
 package govcd
 
 import (
-	"errors"
+	"fmt"
 	"github.com/vmware/go-vcloud-director/v2/types/v56"
 	"net/http"
 )
 
+// DEPRECATED please use GetExternalNetwork function instead
```

Comment:
Why is this function deprecated?

Reply:
Cause it returns `types.ExternalNetworkReference` vCD API type, not `ExternalNetwork` govcd API type